Flat/src/PandaAnalyzer.cc

```diff
@@ -111,8 +111,10 @@ int PandaAnalyzer::Init(TTree *t, TH1D *hweights, TTree *weightNames)
   else if (analysis->fatjet) 
     readlist += {jetname+"CA15Jets", "subjets", jetname+"CA15Subjets","Subjets"};
   
-  if (analysis->recluster || analysis->bjetRegression)
+  if (analysis->recluster || analysis->bjetRegression) {
     readlist.push_back("pfCandidates");
+    readlist.push_back("secondaryVertices");
```

Comment:
This should not be read if `analysis->recluster` is on. Only use it for the regression.